src/arizona_template_compiler.erl

```diff
@@ -344,7 +372,7 @@ compile_tag_attrs([{Name, {expr, ExprLoc, Expr}} | Attrs], Tag, Loc, T, State) -
     [{text, <<$\s, Name/binary, $=, $">>}, Elem, {text, <<$">>}
      | compile_tag_attrs(Attrs, Tag, Loc, T, incr_index(ChangeableCount, State))];
 compile_tag_attrs([], #{is_void := true}, _Loc, T, State) ->
-    [{text, <<$\s, $/, $>>>} | compile_elems(T, State)];
+    [{text, <<$\s, $/, $>>>}, {rest, T, State}];
```

Comment:
It's not the most elegant solution, but that's to not change too much code.
It is used [here](https://github.com/arizona-framework/arizona/pull/189/files#diff-8e7ca3120a4045e0279e67e1275a4fdcbcd21595ae4b702186169c9093ae47cbR315-R316) and [here](https://github.com/arizona-framework/arizona/pull/189/files#diff-8e7ca3120a4045e0279e67e1275a4fdcbcd21595ae4b702186169c9093ae47cbR323-R324).